org.adempiere.base/META-INF/MANIFEST.MF

```diff
@@ -115,6 +115,7 @@ Import-Package: com.github.benmanes.caffeine;version="3.1.8",
  org.antlr.v4.runtime.tree.pattern;version="4.9.2",
  org.antlr.v4.runtime.tree.xpath;version="4.9.2",
  org.apache.activemq;version="5.3.0",
+ org.apache.commons.codec.binary;version="1.16.1",
```

Comment:
This is not needed, java 17 has the same:
java.util.HexFormat.of().formatHex((byte[]) value)

Reply:
hi @uthadehikaru, as per Carlos comments above, please use the build in jdk 17 method instead.